report/renderer/i18n.js

```diff
@@ -169,6 +189,10 @@ export class I18n {
    * @return {string}
    */
   formatDuration(timeInMilliseconds) {
```

Comment:
just realized the comment for this function was wrong

`1d 2h 13m 52s`

we had NPSP all up in there...